# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/efivar/hotfix-clang.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- efivar-38/src/include/defaults.mk.vanilla	2023-05-15 15:51:14.916952734 +0200
+++ efivar-38/src/include/defaults.mk	2023-05-15 15:51:25.400953491 +0200
@@ -45,7 +45,7 @@
 		  -fvisibility=hidden \
 		  $(call family,CFLAGS) \
 		  $(call pkg-config-cflags)
-LDFLAGS_CLANG ?= -rtlib=compiler-rt
+LDFLAGS_CLANG ?= 
 CCLDFLAGS ?=
 LDFLAGS ?=
 override _CCLDFLAGS := $(CCLDFLAGS)
@@ -86,7 +86,7 @@
 HOST_CFLAGS ?= $(CFLAGS) $(call family,HOST_CFLAGS)
 override _HOST_CFLAGS := $(HOST_CFLAGS)
 override HOST_CFLAGS = $(_HOST_CFLAGS)
-HOST_LDFLAGS_CLANG ?= -Wl,--fatal-warnings,-z,relro -rtlib=compiler-rt
+HOST_LDFLAGS_CLANG ?= -Wl,--fatal-warnings,-z,relro
 HOST_LDFLAGS_GCC ?= -Wl,--no-undefined-version
 HOST_LDFLAGS ?=
 HOST_CCLDFLAGS ?=
